app/views/search/index.html.erb

```diff
@@ -100,10 +104,16 @@
                              url_for(@search_request.to_params(order_target: "score"))) %>
         </li>
         <li>
-          <%= link_to_unless(@search_request.order_target == "date",
+          <%= link_to_unless(@search_request.order_target == "last_modified_at",
+                             tag.i(class: "fas fa-clock") + " " +
+                               l(:label_full_text_search_order_target_last_modified_at),
+                             url_for(@search_request.to_params(order_target: "last_modified_at"))) %>
+        </li>
+        <li>
+          <%= link_to_unless(@search_request.order_target == "registered_at",
                              tag.i(class: "fas fa-clock") + " " +
```

Comment:
Can we use different icon?

Reply:
fix: [4e22729](https://github.com/clear-code/redmine_full_text_search/pull/119/commits/4e2272938ce086451177cb751b75de76f7b0a765) Thanks. I changed the icon like the following.
| Before | After |
|---|---|
| <img src=https://github.com/clear-code/redmine_full_text_search/assets/45173523/e249740b-4b3f-4b5e-92c6-f5ff9bdc08ba  width=100%> | <img src=https://github.com/clear-code/redmine_full_text_search/assets/45173523/f89ada6d-c5f5-4031-887a-62066b5514c2  width=100%> |

Reply:
fix: [dbcb629](https://github.com/clear-code/redmine_full_text_search/pull/119/commits/dbcb6299c5bdd91b5484a30bf3ad174a08a0de93)
I couldn't find suitable icons in the clock icon list. I chose the following icons instead:
- Pen icon: Represents 'last_modified_at' as an edit.
- Calendar icon: Symbolizes 'registered_at' as a date-related event.

Reply:
They will be better than the current icons.

Reply:
Sorry, I've forgotten to add the sample image.
<img width="127" alt="Screenshot 2023-12-28 at 7 14 07" src="https://github.com/clear-code/redmine_full_text_search/assets/45173523/f4b5a39a-7a35-4170-ab23-cec13eccf56e">